chart/README.md

```diff
@@ -205,3 +205,9 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 | `tls`                          | "ingress"                                             | ***string*** - See External TLS Termination for details. - *"ingress, external"*                                                                                                                             |
 | `systemDefaultRegistry`        | ""                                                    | ***string*** - private registry to be used for all system Docker images, e.g., [http://registry.example.com/] *Available as of v2.3.0*                                                                       |
 | `useBundledSystemChart`        | false                                                 | ***bool*** - select to use the system-charts packaged with Rancher server. This option is used for air gapped installations.  *Available as of v2.3.0*                                                       |
+| `customLogos.enabled`          | false                                                 | ***bool*** - Enabled [Rancher custom logos](https://github.com/rancher/ui/tree/master/public/assets/images/logos) pvc. K8s storageClass (dynamic pv) or existingClaim (static pv) are required|
+| `customLogos.volumeKind`       | "persistentVolumeClaim"                               | ***string*** - Use persistentVolumeClaim or configMap for custom logos persitence|
+| `customLogos.volumeName`       | ""                                                    | ***string*** - Use an existing volume. Custom logos should be copied to the volume by the user. Optional for persistentVolumeClaim, required for configMap|
+| `customLogos.storageClass`     | ""                                                    | ***string*** - Set custom logos persistentVolumeClaim storage class|
+| `customLogos.accessMode`       | "ReadWriteOnce"                                       | ***string*** - Set custom persistentVolumeClaim access mode|
+| `customLogos.size`             | "10Gi"                                                | ***string*** - Set custom persistentVolumeClaim size|
```

Comment:
this default value does not match the actual default

Reply:
Updated